homer.root/usr/share/clearwater/clearwater-cluster-manager/plugins/homer_cassandra_plugin.py

```diff
@@ -0,0 +1,45 @@
+from metaswitch.clearwater.cluster_manager.plugin_base import \
+    SynchroniserPluginBase
+from metaswitch.clearwater.cluster_manager.plugin_utils import \
+    join_cassandra_cluster, leave_cassandra_cluster
+from metaswitch.clearwater.cluster_manager.alarms import issue_alarm
+from metaswitch.clearwater.cluster_manager import constants
+import logging
+
+_log = logging.getLogger("cassandra_plugin")
+
+
+class HomerCassandraPlugin(SynchroniserPluginBase):
+    def __init__(self, ip):
+        self._ip = ip
+        _log.debug("Raising Cassandra not-clustered alarm")
+        issue_alarm(constants.RAISE_CASSANDRA_NOT_YET_CLUSTERED)
+
```

Comment:
Can you add:

```
def files(self):
    return ["/etc/cassandra/cassandra.yaml"]
```

to each plugin? After discussing with Ellie, I added function to not load multiple plugins that all edit the same file (to allow for colocation).